src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+:root {
+  --active-color: #00acdc;
+
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", "Helvetica Neue", Helvetica, sans-serif;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  width: 1100px;
```

Comment:
1. Please use max-width so that the header does not go beyond the screen borders
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/cb69ee12-36e5-4fbd-be90-79f8bcd57038)
2. Why value is 1100px? I understand that you did this in order to have 50 pixel intend on the sides, but it’s better to make the width as in the task requirments and explicitly set the side paddings
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/b4ec4de5-b0f5-4990-ab90-446768014175)